discovery-installer.spec

```diff
@@ -1,23 +1,24 @@
 Name:           discovery-installer
-Version:        1.8
+Summary:        installer for quipucords server
+
+Version:        1.8.1
 Release:        1%{?dist}
-Summary:        Discovery Application v1.8 Installer
+Epoch:          0
 
 License:        GPLv3
-URL:            https://gihub.com/quipucords/discovery-quadlets
-Source0:        %{name}-%{version}.tar.gz
+URL:            https://github.com/quipucords/discovery-quadlets
+Source0:        %{url}/archive/%{version}.tar.gz
 
 BuildArch:      noarch
 
 Requires:       bash
 
 %description
-This RPM installs the Discovery Application v1.8 Installer on the System.
-The Installer installs and configures Discovery v1.8 to be managed
-and run via systemd quadlet services.
+discovery-installer configures and installs the quipucords server to be
+managed and run via systemd using Podman Quadlet services.
 
 %prep
-%autosetup -n %{name}-%{version}
+%autosetup -n discovery-quadlets-%{version}
```

Comment:
mmmm why not `discovery-installer` (consistency with line 1)?

Reply:
Good question! It has to be the repo name (`discovery-quadlets`) because of the way GitHub creates its source tarballs. I have actually been meaning to ask about renaming this repo, and now's as good a time as any…

Should we rename this project to `discovery-installer` to better reflect its capacity as more than just quadlet configs?

Reply:
I think we shall rename it, then.

Will resolve the conversation here since we have it on slack as well